site/src/pages/RoadmapPage/Header.tsx

```diff
@@ -3,10 +3,11 @@ import { Button, ButtonGroup, Overlay, Popover } from 'react-bootstrap';
 import { ArrowLeftRight, List, Save, Trash } from 'react-bootstrap-icons';
 import { useIsDesktop, useIsMobile, pluralize } from '../../helpers/util';
 import { useAppDispatch } from '../../store/hooks';
-import { clearPlanner, setShowTransfer } from '../../store/slices/roadmapSlice';
+import { setShowClearRoadmapPopup, setShowTransfer } from '../../store/slices/roadmapSlice';
 import './Header.scss';
 import Transfer from './Transfer';
 import RoadmapMultiplan from './RoadmapMultiplan';
+import ClearRoadmapModal from './ClearRoadmapPopup';
```

Comment:
It looks like you updated the import name to `ClearRoadmapModal`, but not the file name?